app/src/main/res/drawable/button_rounded_corners.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android" >
+
+    <item android:state_pressed="true" >
+        <shape android:shape="rectangle"  >
+            <corners android:radius="100dp" />
+            <stroke android:width="1dip" android:color="#5e7974" />
+        </shape>
+    </item>
```

Comment:
Là j'aime pas

Reply:
Please report any issues in the appropriate section.